libs/backend/src/cast_vote_records/export.test.ts

```diff
@@ -511,6 +511,14 @@ test('precinct scanner continuous export', async () => {
     await readCastVoteRecordExportMetadata(exportDirectoryPath)
   ).unsafeUnwrap();
   expect(metadata.arePollsClosed).toEqual(true);
+
+  // confirm batch manifest was properly included
```

Comment:
nit: I think that the following line of code is self-explanatory and doesn't need this comment

Reply:
e9658fa1a